# Good_papers_for_RNAseq

## Feature/marker Selection or identification of gene module
[**Automated cell annotation in scRNA-seq data using unique marker gene sets**](https://www.biorxiv.org/content/10.1101/2024.05.24.595477v1?open_in_browser=true)

[**Feature selection revisited in the single-cell era**](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02544-3)

[**A comparison of marker gene selection methods for single-cell RNA sequencing data**](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03183-0#Tab1)

[**Evaluation of deep learning-based feature selection for single-cell RNA sequencing data analysis**](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03100-x)

[**Supervised discovery of interpretable gene programs from single-cell data**](https://www.nature.com/articles/s41587-023-01940-3)

## Clustering
[**Significance analysis for clustering with single-cell RNA-sequencing data**](https://www.nature.com/articles/s41592-023-01933-9#Sec9)

## Atlas-level analysis
[**Hallmarks of transcriptional intratumour heterogeneity across a thousand tumours**](https://www.nature.com/articles/s41586-023-06130-4)

## Great review
[**The future of rapid and automated single-cell data analysis using reference mapping**](https://www.nature.com/articles/s41586-023-06130-4)

[**Best practices for single-cell analysis across modalities**](https://www.nature.com/articles/s41576-023-00586-w#Fig6)

[**Current best practices in single‐cell RNA‐seq analysis: a tutorial**](https://www.embopress.org/doi/full/10.15252/msb.20188746)

[**Tutorial: guidelines for annotating single-cell transcriptomic maps using automated and manual methods**](https://www.nature.com/articles/s41596-021-00534-0#Sec3)

[**An introduction to spatial transcriptomics for biomedical research**](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-022-01075-1)
