# Good_papers_for_RNAseq

## Feature/marker Selection
[**Automated cell annotation in scRNA-seq data using unique marker gene sets**](https://www.biorxiv.org/content/10.1101/2024.05.24.595477v1?open_in_browser=true)

[**Feature selection revisited in the single-cell era**](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02544-3)

[**A comparison of marker gene selection methods for single-cell RNA sequencing data**](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03183-0#Tab1)

[**Evaluation of deep learning-based feature selection for single-cell RNA sequencing data analysis**](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03100-x)

## Clustering


## Great review
[**The future of rapid and automated single-cell data analysis using reference mapping**](https://www.cell.com/cell/abstract/S0092-8674(24)00301-5#%20)

[**Best practices for single-cell analysis across modalities**](https://www.nature.com/articles/s41576-023-00586-w#Fig6)

[**Current best practices in single‐cell RNA‐seq analysis: a tutorial**](https://www.embopress.org/doi/full/10.15252/msb.20188746)
